gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/util/network/EmergeUploadRequest.kt

```diff
@@ -53,6 +54,7 @@ fun fetchSignedUrl(
   val request: Request = Request.Builder().apply {
     url(url)
     post(body)
+    addHeader("User-Agent", "${SOURCE_GRADLE_PLUGIN}_${BuildConfig.VERSION}")
```

Comment:
Nice, figured you copied it to user-agent here, but we currently just pass this as source to our API: https://github.com/EmergeTools/emerge-android/blob/6fe18887bc31d35373da411656883e843a622090/gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/upload/BaseUploadTask.kt#L153